### ISSUE TITLE
Catalogue of Test Errors and Failures

### ISSUE THREAD

**Issue by djkirkham** [SciTools/cartopy#952]
There are currently several issues with the automated tests. They are listed below.
        
### Python 2.7 with pinned packages numpy=1.7.1, matplotlib=1.3.1, scipy=0.12.0

Failing due to extra attributes in Natural Earth data:
```
[doctest] cartopy.io.shapereader
TestLakes.test_record
TestRivers.test_record
```

Failing due to differences in geometry in Natural Earth data:
```
TestRivers.test_geometry
```

### Python 3.4 with pinned packages numpy=1.8.2, matplotlib=1.3.1, scipy=0.14.0

Several tests are failing due to not being able to find `libgfortran.so.1`. The libgfortran package is installed in the conda environment.

### Python 3.5 "Latest Everything"

This is erroring due to a known error with Python 3.5 (https://bugs.python.org/issue26154)

### Python 2.* "Latest Everything"

In addition to the test failures in the Python 2.7 tests, the following are failing due to image differences, presumably due to using mpl2:

```
test_web_tiles
test_pcolormesh_limited_area_wrap
test_streamplot
```

**Comment by dopplershift** [comment]
See #926 which has CI passing properly.

**Comment by QuLogic** [comment]
Yes, please review #926!

**Comment by djkirkham** [comment]
Closed by #960
